### PR TITLE
client-go: when meet GetStoreMinResolvedTS error just write in debug log

### LINKED ISSUE #44103: Noisy min-resolved-ts logs when the PD version is old
## Bug Report

#43737 introduced a new approach to getting the `min-resolved-ts` from the PD server by HTTP API. However, TiDB will output many noisy logs when the PD version is old due to the 404 HTTP error.

img_v2_37081cb2-fe67-4699-9e8f-11371d2d68ag

### PR DESCRIPTION
**Pull request by HuSharp**
Signed-off-by: husharp <jinhao.hu@pingcap.com>

### What problem does this PR solve?

Issue Number: close #44103 wait for https://github.com/tikv/client-go/pull/814

Problem Summary:

### Check List

Tests 

- [ ] Manual test (add detailed scripts or steps below)

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -294,5 +294,6 @@ replace (
 	github.com/dgrijalva/jwt-go => github.com/form3tech-oss/jwt-go v3.2.6-0.20210809144907-32ab6a8243d7+incompatible
```
Comment (hawkingrei) [inline go.mod:294]: Please remember to run ```make bazel_preparep```.

**Review by JmPotato — APPROVED** [review]

**Review by Defined2014 — APPROVED** [review]

**Pull request merged**